config/application.rb

```diff
@@ -19,7 +19,7 @@
 module Mmt
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 5.2
```

Comment:
I believe that updating this value is equivalent to flipping all of the values in related initializers to true. So, by updating to 5.2 those initializers no longer control anything. If that's accurate, setting them in the environments is only to specifically show that they are on, correct?

Reply:
The initializers were previously removed for this PR,  and only added back last night to see the result on the CI build.    As I have untangled the initialization path, I've been removing the initializer files, one by one,  to ensure that things are operating as before.  I'll finish that process shortly  and will let you know when this is "reviewable".